medicines/search-client/src/lib.rs

```diff
@@ -24,6 +24,17 @@ pub struct AzurePagination {
     pub offset: i32,
 }
 
+pub struct AzureFilterSet {
+    pub boolean_operator: String,
+    pub field_filters: Vec<AzureFieldFilter>,
+}
+
+pub struct AzureFieldFilter {
+    pub field_name: String,
+    pub operator: String,
```

Comment:
It might be nice if `boolean_operator` and `operator` were `enum`s of the different things that are allowed and then we formatted them as strings as needed. Might make the interface a bit safer and nicer to use